docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-intro-con.adoc

```diff
@@ -0,0 +1,8 @@
+[id='using-standalone-perspectives-intro-con']
+= Using standalone perspectives
+
+{CENTRAL} provides different specialized editors for authoring assets which have different formats. As per an asset's format a specific designer is used for editing it. {CENTRAL} has a feature that allows you to use these editors individually. This feature is known as the standalone perspective mode of the editor or simply the _standalone perspectives_.
```

Comment:
Per the IBM Style Guide we shouldn't use "as per". Instead use "According to.." 
However, you could simplify the 1st 2 sentences as follows:
{CENTRAL} provides specialized editors for authoring assets based on the asset's format .

Reply:
Changed as suggested.